packages/web/src/components/basic/ReactiveComponent.js

```diff
@@ -350,18 +350,17 @@ ReactiveComponent.propTypes = {
 ReactiveComponent.componentType = componentTypes.reactiveComponent;
 
 const mapStateToProps = (state, props) => ({
-	aggregations:
-		(state.aggregations[props.componentId] && state.aggregations[props.componentId]) || null,
-	aggregationData: state.compositeAggregations[props.componentId] || [],
-	hits: (state.hits[props.componentId] && state.hits[props.componentId].hits) || [],
+	aggregations: state.aggregations[props.componentId],
+	aggregationData: state.compositeAggregations[props.componentId],
```

Comment:
@SavvyShah Are you handling `null` value for `aggregationData`? The default value of `aggregationData` in `render` method and `onData` callback should be empty array.